executor/logger/constants.go

```diff
@@ -1,24 +1,20 @@
 package logger
 
-import "os"
-
 const (
 	LoggerWorkerQueueSize = 100
 	CloudEventsIdHeader   = "Ce-Id"
 	CloudEventsTypeHeader = "Ce-type"
 	CloudEventsTypeSource = "Ce-source"
 )
 
-// Variable to cache the value of ENV default request logger
-var defaultRequestLoggerEndpointPrefix string = ""
+// Variables to cache the value of ENV default request logger
+var defaultRequestLogger string = ""
+var defaultRequestLoggerEndpointNamespace string = ""
+var defaultRequestLoggerEndpointInModelNamespace string = ""
 
-func GetLoggerDefaultUrl(namespace string) string {
-	if defaultRequestLoggerEndpointPrefix == "" {
-		if value, ok := os.LookupEnv("REQUEST_LOGGER_DEFAULT_ENDPOINT_PREFIX"); ok && value != "" {
-			defaultRequestLoggerEndpointPrefix = value
-		} else {
-			defaultRequestLoggerEndpointPrefix = "http://default-broker."
-		}
+func GetLoggerDefaultUrl() string {
```

Comment:
The method is actually getting the Url not the default - or only the default on fallback? I'm not sure if this way is clearer or how we do in Operator where we get into package var where which means we can do a test:
https://github.com/SeldonIO/seldon-core/blob/9d5eaa74b8b78b6a316149adf995505864934732/operator/controllers/seldondeployment_engine.go#L55

Reply:
Yeah I think that's better. Also better to be consistent. Have changed it do that now.